rdt/transformers/base.py

```diff
@@ -109,6 +109,16 @@ def reset_randomization(self):
             'reverse_transform': np.random.RandomState(self.random_seed + 1)
         }
 
+    @property
+    def model_missing_values(self):
```

Comment:
the categorical transformers don't have this property but would get it from this right? Should we just put it in the ones that don't have it? I think it would crash on the `FrequencyEncoder` since there is no `self.missing_value_generation`